src/editor/src/routes.js

```diff
@@ -0,0 +1,44 @@
+import fs from '../../lib/fs';
+const sh = new fs.Shell();
+
+// Returns the URL's paths
+function splitURL(urlPath) {
+    return urlPath.substr(6).split('/');
+}
+
+const editRegex = new RegExp('\/edit(\/.*)/');
+
+export default {
+    init: (workbox) => {
+        // @ts-ignore
+        workbox.routing.registerRoute(
+            editRegex,
+            async ({ url, event }) => {
+                const path = url.pathname.match(editRegex)[1];
+                const paths = splitURL(url.pathname);
+
+                let folder = file = redirectURL = "";
+                
+                // Untested Code!!!!
```

Comment:
Please verify whether this code works and remove the comment.

Reply:
Cannot Test out the routes since the service worker cannot load with the current build. Once that is solved i'll be able to test.